Maddori.Apple/Maddori.Apple/Screen/Home/Home/TeamManageViewController.swift

```diff
@@ -44,10 +53,22 @@ final class TeamManageViewController: BaseViewController {
     
     // MARK: - life cycle
     
+    init(teamId: Int) {
+        self.currentTeamId = teamId
+        super.init()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
p2;

기존의 저희가 사용하던 방식인
```suggestion
        nil
```
로 처리하는게 어떨까요?

Reply:
[561f6f9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/328/commits/561f6f9b1a4a93b5fe611118433fa737418a4e15)